Tests/Tensor/test_Tensor_adjointH.cpp

```diff
@@ -76,16 +76,17 @@ void test_adjoint(SizeTypes ... Ns){
     for(NSL::size_t d1 = 0; d1 < sizeof...(Ns); ++d1){
         for(NSL::size_t d2 = 0; d2 < sizeof...(Ns); ++d2){
 			// in this case A and B change every iteration
-            B.transpose(d1,d2).conj();
+            // .T creates a copy of A
+            NSL::Tensor<Type> B = A.T(d1,d2).conj();
             
             NSL::Tensor<Type> C = A.adjoint(d1,d2);
 
 			// check that data agrees
             REQUIRE((B == C).all());
-			// check that the correct tensors share/ not share their data
-            REQUIRE( A.data() == addr_bak );
-            REQUIRE( C.data() == addr_bak );
-            REQUIRE( B.data() != addr_bak );
+            REQUIRE((A == C).all());
+            REQUIRE((A == B).all());
```

Comment:
I don't understand these.  If B and C are conj of A, how can they be equal if A is complex?  Also if B and C are transpose of A how can they be equal if A is not symmetric on the (d1, d2) axes?

Reply:
Because B,C are views and A's data is changed internally.

Reply:
Can you put a comment that reminds us that A.adjoint is mutating?

Reply:
Done with f5f69f70ed6eaed6d87b8fa72d0075b8f72b6722